cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -60,6 +60,76 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
         });
       });
     }
+    // This test is being temporarily throttled to the service 'news'.
+    if (service === 'news') {
+      it('should have a visible play button and duration', () => {
+        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+          ({ body }) => {
+            const media = getBlockData('video', body);
+            if (media) {
+              const aresMediaBlocks = media.model.blocks[1].model.blocks[0];
+              const { durationISO8601 } = aresMediaBlocks.model.versions[0];
+              cy.get('div[class^="StyledVideoContainer"]')
+                .then(() => {
+                  cy.get('iframe[class^="i-amphtml-fill-content"]').then(
+                    $iframe => {
+                      cy.wrap($iframe.prop('contentWindow'), {
+                        // `timeout` only applies to the methods chained below.
+                        // `its()` benefits from this, and will wait up to 8s
+                        // for the mediaPlayer instance to become available.
+                        timeout: 8000,
+                      })
+                        .its('embeddedMedia.playerInstances.mediaPlayer')
+                        .invoke('currentTime')
+                        .should('be.gt', 0);
+                    },
+                  );
+                })
+                .within(() => {
```

Comment:
I think this needs a callback defined, even if its not used

```suggestion
                .within((el) => {
```

source: https://docs.cypress.io/api/commands/within.html#Syntax